engine/apps/api/views/user.py

```diff
@@ -350,6 +352,13 @@ def get_telegram_verification_code(self, request, pk):
 
         return Response({"telegram_code": str(new_code.uuid), "bot_link": bot_link}, status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=["post"])
+    def unlink_slack(self, request, pk):
+        user = self.get_object()
+        user.slack_user_identity = None
```

Comment:
Should we also delete the `SlackUserIdentity` instance in case there are no users associated with it?

Reply:
I based this on the existing reset slack process (`ResetSlackView`), but also I think we don't want to lose track of any potential data (messages, etc) associated to the slack user?

Reply:
Makes sense 👍